File: Mensuration/Square and Cube/Area/AoC.java
//Area of a Circle
import java.util.Scanner;
public class AoC{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter Radius : ");
        int r=sc.nextInt();
        sc.close();
        double area=Math.pow(r,2)*Math.PI;
        System.out.println("Area : "+area);
    }
}

File: Problem Statements/Mensuration/Circle and Sphere/Area/AoC.cpp
//Area of a Circle
#include<iostream>
#include<math.h>
using namespace std;
#define Pi 3.14

void main(){
    int r;
    float area=0.0;
    cout<<"Enter the Radius : ";
    cin>>r;
    area = Pi *pow(r,2);
    cout<<"Area : "<<area;
}